src/views/InteractiveMap/CardDialogDataDriven.vue

```diff
@@ -3,7 +3,57 @@
     <v-card class="pa-2">
       <v-form>
         <v-container>
-          <p class="headline">Modify data-driven simulation</p>
+          <v-layout row wrap>
+            <v-flex xs6>
+              <p class="headline">Modify data-driven simulation</p>
+            </v-flex>
+            <v-flex xs6>
+              <p class="text-xs-right subheading mb-0 mt-2 font-italic">
+                Simulation status:
+                <span v-if="card.solverStatus">
+                  <span v-if="card.solverStatus == 'optimal'">Successful</span>
+                  <span v-else-if="card.solverStatus == 'infeasible'"
+                    >Infeasible</span
+                  >
+                </span>
+                <span v-else-if="card.sampleErrors.length > 0"
+                  >Can not integrate data</span
```

Comment:
(moving these many conditions into e.g. getStatusMessage method (or computed) would be easier to read (+ `<v-progress-circular v-if="modfying||simulating">`))